src/apis/abusiveexperiencereport/v1.ts

```diff
@@ -167,6 +167,55 @@ export namespace abusiveexperiencereport_v1 {
     /**
      * abusiveexperiencereport.sites.get
      * @desc Gets a site's Abusive Experience Report summary.
+     * @example
+     * // Before running the sample:
+     * // - Enable the API at:
+     * //   https://console.developers.google.com/apis/api/abusiveexperiencereport
```

Comment:
This link did not work for me. Not sure if it's something with my account, or the link is wrong. Just FYI.

Reply:
Ugh.  Updating this to append `.googleapis.com` (not needed in all of them)